### DISCUSSION TITLE
Lack Title with the same name as the file name for diary notes

### DISCUSSION THREAD

**Discussion by zelenukhin** [GitJournal/GitJournal#691] [Feature Requests]
Lack Title with the same name as the file name for diary notes (example 2021-07-02) as a result when the Title mode is enabled, I do not see a diary notes in the list of all notes.
Please add automatic Title creation for diary entries. The title value is filled in based on the New Note Filename settings. In other words, Title equals File Name for diary notes.

vhanda's note:
This would mean adding a new option for new Journal entries where an H1 with the date is added.

**Now:**
![image0](https://user-images.githubusercontent.com/5934321/124593799-bd11f300-de67-11eb-9d57-7cb1d638754c.jpg)
![image1](https://user-images.githubusercontent.com/5934321/124593772-b7b4a880-de67-11eb-9781-b9bd8b314373.jpg)
![image2](https://user-images.githubusercontent.com/5934321/124593826-c602c480-de67-11eb-9fa5-91cc7e444f74.jpg)

**Wishing:**
![photo_2021-07-06_14-41-20](https://user-images.githubusercontent.com/5934321/124594479-812b5d80-de68-11eb-8295-d3c65877f632.jpg)
![photo_2021-07-06_14-41-14](https://user-images.githubusercontent.com/5934321/124594493-84bee480-de68-11eb-8696-30fe5ebfac1d.jpg)
![photo_2021-07-06_14-44-30](https://user-images.githubusercontent.com/5934321/124594650-bafc6400-de68-11eb-8f34-dce382c35d03.jpg)

**Comment by zelenukhin** [comment]
The Title or FileName mode instead of Auto Generated Title partially solves the problem. Nevertheless, I have to add Title manually.

https://user-images.githubusercontent.com/5934321/124595542-d2881c80-de69-11eb-93e2-85ff120effa0.MP4

**Comment by vHanda** [comment]
What do you mean by title in this case? Do you mean the h1?

**Comment by zelenukhin** [comment]
Yes - H1. 
GitJournal in Markdown editor mode - displays first H1 as Title

![photo_2021-07-06_15-03-01](https://user-images.githubusercontent.com/5934321/124596713-47a82180-de6b-11eb-8242-6bd666c45e23.jpg)

**Comment by vHanda** [comment]
Hmm. Okay, this would need to be a new option.

**Comment by zelenukhin** [comment]
Cool. I'll be waiting for an update!
